docs/news.rst

```diff
@@ -67,11 +72,13 @@ The following parsers have been added:
 * Maven2 *libxml*
 * Moose *perl basesd subperser*
 * M4
+* Nsis
 * ObjectiveC
 * Passwd *optlib*
 * PuppetManifest *optlib*
 * Perl6
 * Pod *optlib*
+* Powershell
```

Comment:
Very minor issue. Powershell should be PowerShell.

Reply:
Fixed, including a few more capitalization problems.

Reply:
Thank you.